mlrun/config.py

```diff
@@ -650,6 +650,7 @@
         # used for igz client when emitting events
         "access_key": "",
     },
+    "external_grafana_url": "",
```

Comment:
i'm not sure "external" is very informative - in both cases (iguazio / ce) grafana is some pod in the k8s.
why not just `grafana_url` (that happens to be configured in ce case, but possibly in other cases)